app/views/groups/attachments/index.html.erb

```diff
@@ -0,0 +1,81 @@
+<%= turbo_refreshes_with method: :morph, scroll: :preserve %>
+
+<%= turbo_frame_tag "attachment_modal" %>
+
+<div class="fixed-table-component">
+  <%= render Viral::PageHeaderComponent.new(
+    title: t(".title"),
+    subtitle: t(".subtitle", group_name: @group.name),
+  ) do |component| %>
+    <%= component.with_buttons do %>
+    <% if allowed_to?(:create_attachment?, @group) %>
+      <%= link_to t(".upload_files"),
+          new_group_attachment_path(id: @group.id),
+          data: {
+            turbo_frame: "attachment_modal",
+            turbo_stream: true,
+          },
+          class: "
+            inline-flex
+            items-center
+            justify-center
+            w-1/2
+            text-sm
+            border
+            rounded-md
+            cursor-pointer
+            sm:w-auto
+            focus:z-10
+            px-5
+            py-2.5
+            bg-white
+            text-slate-900
+            border-slate-200
+            focus:outline-none
+            hover:bg-slate-100
+            hover:text-slate-950
+            focus:ring-slate-200
+            dark:focus:ring-slate-700
+            dark:bg-slate-800
+            dark:text-slate-400
+            dark:border-slate-600
+            dark:hover:text-white
+            dark:hover:bg-slate-700
+          " %>
+      <% end %>
+    <% end %>
+  <% end %>
+
+  <div class="flow-root">
+    <div class="flex flex-row-reverse items-center mb-4 space-x-2">
+      <%= search_form_for @q, url: group_attachments_path(@group), html: { "data-controller": "filters" } do |f| %>
+        <%= hidden_field_tag :limit, @pagy.limit %>
+        <%= f.hidden_field :s, value: "#{@q.sorts[0].name} #{@q.sorts[0].dir}" %>
+        <%= f.hidden_field :format, value: "turbo_stream" %>
```

Comment:
**nitpick (blocking):** Can you remove this form input as it is not being used, it also needs to be removed from the projects attachments index.

Reply:
Fixed in c563452c2d2275c825749673841e210d9de490fa